magic/fetcher.py

```diff
@@ -124,15 +124,15 @@ def legal_cards(force: bool = False, season: str = None) -> List[str]:
         url = 'legal_cards.txt'
     else:
         url = '{season}_legal_cards.txt'.format(season=season)
-    encoding = 'utf-8' if season != 'EMN' else 'latin-1' # EMN was encoded weirdly.
+    encoding = 'utf-8'
     cached_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'legal_cards')
     if os.path.exists(os.path.join(cached_path, url)):
         h = open(os.path.join(cached_path, url), encoding=encoding)
         legal = h.readlines()
         h.close()
         return [l.strip() for l in legal]
 
-    url = 'http://pdmtgo.com/' + url
+    url = 'https://pennydreadfulmtg.github.io/' + url
```

Comment:
What's the benefit of using the github address? You're staging there?

Reply:
Yeah, and the rotation script automatically pushes there, whereas S3 needs manual intervention.

It means our internal tools will be faster and require less human intervention.